conda/environments/rmm_dev_cuda10.0.yml

```diff
@@ -0,0 +1,21 @@
+name: rmm_dev
```

Comment:
I don't use a separate env for RMM and cuDF development. I'm not sure if this is necessary.

Reply:
I felt for cohesion we could define a minimal environment for someone who only cares about RMM development. That being said this needs to be trimmed down more as it includes a ton of packages related to docs building for cuDF.

Reply:
Are you planning to trim this down for this PR?